x-pack/legacy/plugins/siem/server/lib/detection_engine/alerts/utils.ts

```diff
@@ -54,39 +55,52 @@ export const singleBulkIndex = async (
     },
     buildBulkBody(doc, params),
   ]);
+  const time1 = performance.now();
   const firstResult = await service.callCluster('bulk', {
-    refresh: true,
+    index: '.siem-signals-10-01-2019',
```

Comment:
Hard coded signals output we want to avoid. I would use:

```ts
process.env.SIGNALS_INDEX || '.siem-signals-10-01-2019'
```

For now until the ticket for signal index creation is completed.

Reply:
Ah shoot. Thanks.